src/briefcase/commands/create.py

```diff
@@ -252,16 +252,19 @@ def generate_app_template(self, app: BaseConfig):
         # use a default template.
         if app.template is None:
             app.template = self.app_template_url
+        if app.template_branch is None:
+            app.template_branch = self.python_version_tag
```

Comment:
Something that occurred to me on a re-review - should this branch be executed if you define a custom template URL?  

I can see arguments both ways. On the one hand, requiring developers to use a "3.8" branch for their 3.8 custom template makes sense. This would technically be backwards incompatible, but I'm not *overly* concerned about that. 

On the other hand, the most likely use case I can imagine is "I made a custom template for my project, and I've only got one (master) branch." Requiring that users manually specify a branch of "master" will be a bit of a PITA, given the default behaviours of git.

Thoughts?

Reply:
I prefer the user to create his/her custom template repository with the 3.6/3.7/3.8 branches by default and to specify explicitly otherwise.

ie. by default the branches should be as the python version, but the user can define if it's not the case.

Reply:
Fair enough. I suspect custom template repositories are going to be an edge case (or, at least, I *hope* they will be) - so it won't hurt too much to force projects that need a custom template to version them, and explicitly opt *out* if they don't want to use versioning.